test/fixtures/simple_example/variables.tf

```
@@ -14,7 +14,37 @@
  * limitations under the License.
  */
 
-variable "project_id" {
-  description = "The ID of the project in which to provision resources."
-  type        = "string"
+variable "parent_id" {
+  description = "Id of the resource under which the folder will be placed."
+  default = "65552901371"
+}
+
+variable "parent_type" {
+  description = "Type of the parent reosurce, defaults to organization."
+  default     = "folder"
+}
+
+variable "names" {
+  description = "Folder names."
+  default     = [
+    "dev",
+    "staging",
+    "production",
+  ]
+}
+
+variable "per_folder_admins" {
+  description = "List of comma delimited IAM-style members per folder who will get extended permissions."
```

Lists are always comma delimited.

```suggestion
  description = "List of IAM-style members per folder who will get extended permissions."
```